### PR TITLE
OPSEXP-2275: Set search enterprise as the default search engine in compose

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2275

### CODE REVIEW

**Comment by alxgomz** [comment]
Tests need to be fixed (moved to DTAS?)

File: docker-compose/solr6-override-docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  # Disable the default elastcisearch service
+  search:
+    image: alpine:latest
+    command: "true"
+  search-reindexing:
+    image: alpine:latest
+    command: "true"
+  elasticsearch:
+    image: alpine:latest
+    command: "true"
```
Comment (gionn) [inline docker-compose/solr6-override-docker-compose.yml:59]: `busybox:musl` is less than 1M :D

File: docker-compose/23.N-docker-compose.yml [hunk 2]
```diff
@@ -69,7 +77,7 @@ services:
         http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file
     ports:
       - "8095:8095"
-    links:
```
Comment (gionn) [inline docker-compose/23.N-docker-compose.yml:72]: 💘

Comment (gionn) [reply docker-compose/23.N-docker-compose.yml:72]: but for consistency I would raise another PR to do the same for all the composes we have (epic OPSEXP-2256)

File: docker-compose/solr6-override-docker-compose.yml
Comment (gionn) [inline docker-compose/solr6-override-docker-compose.yml:1]: maybe we should document here how this file is meant to be used and/or links to the docs

File: docs/docker-compose/README.md [hunk 3]
```diff
@@ -12,61 +13,88 @@ Using the Community compose file will deploy the following system:
...
+   * [http://<machine_ip>:8080/alfresco/service/enterprise/admin/admin-license](http://localhost:8080/alfresco/service/enterprise/admin/admin-license) (`<machine_ip>` will usually just be `localhost`)
+   * Default username and password is `admin`
    * See [Uploading a new license](https://docs.alfresco.com/content-services/latest/admin/license/#uploadlicense) for more details
 
-### Search Enterprise 3.0 (Elasticsearch)
+### Choosing a search engine
 
-You can provision the latest Alfresco with the new search subsystem based on Elasticsearch rather than solr with:
+Alfresco comes either with Solr or Elasticsearch as a Full Text Search engine.
+By default ACS enterprise flavor 23 and above are shiped with Elasticsearch.
```
Comment (gionn) [inline docs/docker-compose/README.md:65]:
```suggestion
By default ACS enterprise flavor 23 and above are shipped with Elasticsearch.
```

**Review by gionn — CHANGES_REQUESTED** [review]

File: docker-compose/solr6-override-docker-compose.yml
Comment (alxgomz) [reply docker-compose/solr6-override-docker-compose.yml:1]: just added a link to the updated doc

File: docker-compose/23.N-docker-compose.yml (hunk 2 above)
Comment (alxgomz) [reply docker-compose/23.N-docker-compose.yml:72]: will create a new ticket in this epic

**Review by gionn — APPROVED** [review]

**Pull request merged**